py-rse/errors.Rmd

```diff
@@ -1,11 +1,14 @@
 # Error Handling {#errors}
 
+We live in an imperfect world.
 People will give our programs options that aren't supported
 or ask those programs to read files that don't exist.
 Our code will also inevitably contain bugs,
 so we should plan from the start to catch and handle errors.
-
-Your Zipf's Law project should now include:
+In this chapter,
+we will explore how errors are represented in programs
+and what we should do with them.
+The Zipf's Law project should now include:
```

Comment:
In the summary of the project tree `mymodule.py` needs to be updated to `utilities.py` and `rcparams.yml` to `plotparams.yml`.

(We've failed to update `mymodule.py` in previous chapters too.)

Reply:
Good catch - amended in all files.